src/main/java/cc/polyfrost/vanillahud/mixin/GuiIngameForgeMixin.java

```diff
@@ -0,0 +1,16 @@
+package cc.polyfrost.vanillahud.mixin;
+
+import net.minecraftforge.client.GuiIngameForge;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(value = GuiIngameForge.class, remap = false)
```

Comment:
can you make the priority of this mixin Integer.MIN_VALUE or MAX_VALUE ( I don't remember which one made it so it injects last )

Reply:
edit: its MAX_VALUE (its probably better to set this to like 9000 or whatever though to keep space for other mods or something yeah good)

Reply:
I did try changing the priority to Max, Min, and 9000 and checked if that would resolve Hytils' hide checks not taking effect when using this mod. So far, this change has not resolved the issue.